### ISSUE TITLE
Improve redability when acquiring parent directory path

### ISSUE THREAD

**Issue by tiagohonorato** [avocado-framework/avocado#4300]
Maybe we could improve the readability on occasions where we obtain the parent path using: 

``` python
BASEDIR = os.path.dirname(os.path.abspath(__file__))
BASEDIR = os.path.abspath(os.path.join(BASEDIR, os.path.pardir))
```
Based on the discussion in #4289 and looking on how the `os.path` module is used in the project, one possible alternative could be:

``` python
BASEDIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
The above is also used in `selftests/check_tmp_dirs`

**Comment by willianrampazzo** [comment]
Hi @tiagohonorato, thanks for bringing this discussion.

In fact, IMHO, if the final result is the same and it does not affect performance, which I don't think it does, I don't see a problem in changing that.

I think one point brought by @beraldoleal that is worth discussing is the import of the functions instead of the modules, just like on his example:

```python
from os.path import abspath, dirname

BASEDIR = dirname(dirname(abspath(__file__))))
```

If we agree that it keeps the readability, I also see a benefit here. Some people may argue that importing the function will always have the extra step of checking the import to know its source. The important step here is to discuss and decide which approach we will take in the project.

**Comment by beraldoleal** [comment]
I agree with @willianrampazzo , we can keep like that. But if you would like to change, IMO we should try the suggestion that I mentioned on #4289 and @willianrampazzo posted here. But this would need to change other parts of the module. So yes, it is welcome but I don't think that has a high priority, right now.

**Comment by tiagohonorato** [comment]
Hi @willianrampazzo and @beraldoleal, thanks for bringing that up as well. 

I guess is like what @willianrampazzo said, although importing the functions is definitely less confusing to read, it also becomes less clear about what it does. But I agree with you all, maybe that is something worth trying, definitely low priority though, and it may require some changes outside `os.path` with variable names, for instance.
